### PR TITLE
Remove support for Python 3.6

### PR DESCRIPTION
**Pull request by eccles**
Problem:
Python 3.6 is EOL December 2021. Additionally we are unable
to use an increasing number of dependencies as they withdraw
support for 3.6. Subsequent work involving using swagger
json definitions to generate dataclasses is restricted to
3.7 and later.

Solution:
Remove all support for Python 3.6 and default to 3.7.

Signed-off-by: Paul Hewlett <phewlett76@gmail.com>

### CODE REVIEW

File: unittests/__init__.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
 """
 Unit tests
 """
+import unittest
+
+# Hides Docstring from Verbosis mode
+unittest.TestCase.shortDescription = lambda x: None
```
Comment (JAG-UK) [inline unittests/__init__.py:7]: "verbose"?

File: unittests/__init__.py [hunk 2]
```diff
@@ -1,3 +1,7 @@
 """
 Unit tests
 """
+import unittest
+
+# Hides Docstring from Verbosis mode
```
Comment (JAG-UK) [inline unittests/__init__.py:6]: "verbose"?

**Review by JAG-UK — CHANGES_REQUESTED** [review]

File: unittests/__init__.py (hunk 1 above)
Comment (eccles) [reply unittests/__init__.py:7]: fixed

File: unittests/__init__.py (hunk 2 above)
Comment (eccles) [reply unittests/__init__.py]: fixed

**Pull request closed without merging**